src/main/java/mate/academy/rickandmorty/controller/ApiController.java

```diff
@@ -0,0 +1,18 @@
+package mate.academy.rickandmorty.controller;
+
+import lombok.RequiredArgsConstructor;
+import mate.academy.rickandmorty.model.CharacterEntity;
+import mate.academy.rickandmorty.service.RickAndMortyClient;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequiredArgsConstructor
```

Comment:
better endpoint mapping
```suggestion
@RequiredArgsConstructor
@RequestMapping("/characters")
```